docs/resources/apig_certificate.md

```diff
@@ -0,0 +1,150 @@
+---
+subcategory: "API Gateway (Dedicated APIG)"
+---
+
+# huaweicloud_apig_certificate
+
+Manages an APIG SSL certificate resource within HuaweiCloud.
+
+## Example Usage
+
+### Manages a global SSL certificate
+
+```hcl
+variable "certificate_name" {}
+variable "certificate_content" {
+  type    = string
+  default = "'-----BEGIN CERTIFICATE-----THIS IS YOUR CERT CONTENT-----END CERTIFICATE-----'"
+}
+variable "certificate_private_key" {
+  type    = string
+  default = "'-----BEGIN PRIVATE KEY-----THIS IS YOUR PRIVATE KEY-----END PRIVATE KEY-----'"
+}
+
+resource "huaweicloud_apig_application" "test" {
```

Comment:
The resource name is `huaweicloud_apig_certificate`. Please check the other parts

Reply:
done.